### ISSUE TITLE
Send webmention to user's profile url when mentioned

### ISSUE THREAD

**Issue by EdwardHinkle** [microdotblog/issues#83]
Currently, I’m getting micro.blog webmentions by running a proxy for myself that pings my user’s mentions feed every couple of minutes. This wouldn’t be need for me (or anyone else) if micro.blog sent a webmention with a target of the user’s profile url.

The microformats might have to be tweaked a little to get it to work.

The scenario is this:

  * Aaron Parecki writes a post that @mentions my micro.blog username, creating a micro.blog mention.
  * Micro.blog sees that and adds the post to my mentions feed
  * Micro.blog checks the URL I have associated with my micro.blog profile (https://eddiehinkle.com). This also means https://eddiehinkle.com would need to be included in the post url’s h-entry e-content somewhere, whether or not it actually displays on the page
  * Micro.blog would send a webmention to my profile url (https://eddiehinkle.com), allowing my site and webmention endpoint to receive the mention without having to ping micro.blog.

**Comment by aaronpk** [comment]
I was just thinking about this the other day! I wasn't sure whether you'd want to hyperlink the @-mention to the user's canonical domain or keep it as the micro.blog profile link, but eddie's idea of adding an invisible link inside the h-entry would work too!

**Comment by gerwitz** [comment]
This might be generalized to "send webmentions sent to my micro.blog profile URL to my personal URL," I think.

Could that be as simple as adding a Webmentions endpoint link to the profile page? If not, maybe an HTTP redirect?

**Comment by ghost** [comment]
Was this added in either of the following updates: [Webmention improvements](https://www.manton.org/2018/10/09/webmention-improvements-on.html); [Following blogs](https://www.manton.org/2019/05/16/following-other-blogs.html)?

**Comment by gerwitz** [comment]
> Was this added in either of the following updates: [Webmention improvements](https://www.manton.org/2018/10/09/webmention-improvements-on.html); [Following blogs](https://www.manton.org/2019/05/16/following-other-blogs.html)?

I don’t think so, since the “root” post of a conversation is still the micro.blog URL.

**Comment by ghost** [comment]
Ah. Understood. 👌🏻
